mlrun/package/packagers/python_standard_library_packagers.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import datetime
+import os
+import pathlib
+import tarfile
+import zipfile
+from enum import Enum
+
+from mlrun.artifacts import Artifact
+from mlrun.datastore import DataItem
+from mlrun.errors import MLRunInvalidArgumentError
+
+from ..common import ArtifactType
+from .default_packager import DefaultPackager
+
+
+# builtins packagers:
+class IntPackager(DefaultPackager):
+    PACKABLE_OBJECT_TYPE = int
+    DEFAULT_ARTIFACT_TYPE = ArtifactType.RESULT
+
+
+class FloatPackager(DefaultPackager):
+    PACKABLE_OBJECT_TYPE = float
+    DEFAULT_ARTIFACT_TYPE = ArtifactType.RESULT
+
+
+class BytesPackager(DefaultPackager):
+    PACKABLE_OBJECT_TYPE = bytes
+    DEFAULT_ARTIFACT_TYPE = ArtifactType.RESULT
+
+
+class StrPackager(DefaultPackager):
+    PACKABLE_OBJECT_TYPE = str
+    DEFAULT_ARTIFACT_TYPE = ArtifactType.RESULT
+
+    class ArchiveFormats(Enum):
+        ZIP = "zip"
+        TAR_GZ = "tar.gz"
+
+    @classmethod
+    def pack_file(cls, obj: str, key: str):
+        # Verify the path is of an existing file:
+        if not os.path.exists(obj):
+            raise MLRunInvalidArgumentError(f"The given path do not exist: '{obj}'")
+        if not os.path.isfile(obj):
+            raise MLRunInvalidArgumentError(f"The given path is not a file: '{obj}'")
+
+        # Create the artifact:
+        artifact = Artifact(key=key, src_path=os.path.abspath(obj))
+
+        return artifact, None
+
+    @classmethod
+    def pack_directory(cls, obj: str, key: str, fmt: str = "zip"):
+        pass
+
+    @classmethod
+    def unpack_file(
+        cls,
+        data_item: DataItem,
+    ) -> str:
+        return data_item.local()
+
+    @classmethod
+    def unpack_directory(cls, data_item: DataItem, fmt: str = None):
+        archive_file = data_item.local()
```

Comment:
I would prefer to have a full parameter name and not a shorted one, but if you feel like it's better to stay with the `fmt` please add docstring detailing what is it for and what option there are
```suggestion
    @classmethod
    def unpack_directory(cls, data_item: DataItem, archive_format: str = None):
        archive_file = data_item.local()
```

Reply:
All of the packagers are yet to be implemented. The current ones are just for idea. I waited for full review before I proceed to implement them.